bin/run_data_task.py

```diff
@@ -47,7 +47,7 @@
 def main():
     args = parser.parse_args()
 
-    with bootstrap(args.config_file) as env:
+    with bootstrap(args.config_file) as env:  # noqa: PLR1702
```

Comment:
The linter has a point, this is too nested but I'm not going to refactor it right now.